packages/tenant-readmodel-writer/src/tenantConsumerServiceV1.ts

```diff
@@ -29,9 +27,8 @@ export async function handleMessage(
         { upsert: true }
       );
     })
-    .with({ type: "TenantDeleted" }, async (_msg) => {
-      logger.info("TODO");
-    })
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    .with({ type: "TenantDeleted" }, async (_msg) => {})
```

Comment:
`TenantDeleted` should delete the tenant

Reply:
This change came in when I merged the main, so it was already present on a previous branch, which is why it wasn't implemented on this one too